sys-libs/libatomic_ops/libatomic_ops-7.6.2.recipe

```diff
@@ -0,0 +1,84 @@
+SUMMARY="An atomic memory update operations portable implementation"
+DESCRIPTION="Semi-portable access to hardware-provided atomic memory update \
+operations on a number architectures."
+HOMEPAGE="https://github.com/ivmai/libatomic_ops"
+COPYRIGHT="1991-1994 Xerox Corporation
+	1996-1999 Silicon Graphics
+	2003-2011 Hewlett-Packard Development Company, L.P.
+	2005,2007 Thiemo Seufer
```

Comment:
I forgot to spot the missing space here, but since there are a few other changes, could you fix this? Thanks!
```diff
-	2005,2007 Thiemo Seufer
+	2005, 2007 Thiemo Seufer
```
(Well, although we have several recipes with a similar missing space, most recipes have that space, so it's better to add it here too.)